packages/caip/src/adapters/banxa/index.ts

```diff
@@ -47,3 +50,29 @@ export const getSupportedBanxaAssets = () =>
     CAIP19,
     ticker
   }))
+
+/**
+ * map CAIP2s to Banxa blockchain codes (ETH, BTC, COSMOS),
+ * since some Banxa assets could be on multiple chains and their default
+ * chain won't be exactly the same as ours.
+ */
+const caip2ToBanxaBlockchainCodeMap: Record<CAIP2, string> = {
```

Comment:
We might even want to `as const` this if this is compatible with the `Record<CAIP2, string>` type

Reply:
agree with as const, and please rename to `chainIdToBanxaBlockchainCodeMap`